cmd/antrea-agent/agent.go

```diff
@@ -160,7 +165,16 @@ func run(o *Options) error {
 		wireguardConfig,
 		networkReadyCh,
 		stopCh,
-		features.DefaultFeatureGate.Enabled(features.AntreaProxy))
+		features.DefaultFeatureGate.Enabled(features.AntreaProxy),
+		enableBridge)
+	if enableBridge {
+		// Restore network config before shutdown. ovsdbConnection must be alive when restore.
+		defer agentInitializer.RestoreOVSBridge()
+		// remove "flow-restore-wait" config immediately when enableBridge=true to enable Node network.
+		if err := agentInitializer.FlowRestoreComplete(); err != nil {
```

Comment:
Does it cause any trouble if we wait for a while to enable Node network? Asking because this would cause container connection issue if enable forwarding before flows are restored (that's why it's called FlowRestoreComplete).

Reply:
Yes. After the uplink bridged to OVS, the Node network doesn't work until FlowRestoreComplete(), which will cause agent cannot access K8s apiserver and fail the initialize.

Reply:
The first function affected by this is i.initNodeLocalConfig(), thus we must run FlowRestoreComplete() before it.

Reply:
@wenyingd Could you explain how Windows solve this issue since no flow-restore-wait in Windows?

Reply:
Windows doesn't use flow-restore-wait. If the uplink is still attached on OVS bridge, and no flows on OVS Bridge, Windows host is actually not acceptable. I think this should be more critical than container connection issue.